config/viper_test.go

```diff
@@ -2,19 +2,17 @@ package config
 
 import (
 	"bytes"
+	"os"
 	"testing"
 
 	"github.com/spf13/viper"
 	"github.com/stretchr/testify/require"
-
-	_ "embed"
 )
 
-//go:embed test_files/config.json
-var configFile string
-
 func TestBuildConfig(t *testing.T) {
 	v := viper.New()
+	cfgBytes, err := os.ReadFile("../sample-relayer-config.json")
```

Comment:
Let's check the error here. This is also the cause of the failing lint job.

Reply:
Sorry my mind was elsewhere. It should be ok now

Reply:
No worries! Thanks for the fix.